docs/vsphere-golden-image.md

```diff
@@ -4,13 +4,17 @@ This guide describes the thought process of creating a Windows virtual machine b
 
 ## 1. Select a compatible Windows Server version
 
-Currently, the Windows Machine Config Operator (WMCO) stable version only supports Windows Server Semi-Annual
-Channel (SAC): Windows Server 2004, that includes the patch [KB4565351](https://support.microsoft.com/en-us/help/4565351/windows-10-update-kb4565351), 
-required by the operating system to allow using the [hybrid OVN Kubernetes networking with a 
-custom VXLAN port](setup-hybrid-OVNKubernetes-cluster.md#vSphere) feature.
-
-*Please note that Windows Server Long-Term Servicing Channel (LTSC): Windows Server 1809 cannot be used, since 
-the patch is not available.*
+Currently, the Windows Machine Config Operator (WMCO) stable version supports:
+<ul>
+    <li>Windows Server 2022</li>
+    <li>Windows Server 20H2</li>
```

Comment:
What about just adding a pointer to [Supported Windows Server versions](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/wmco-prerequisites.md#supported-windows-server-versions)? It Makes things simpler and avoid content duplication.

Reply:
I think its worth keeping here explicitly given [the extra explanation](https://github.com/openshift/windows-machine-config-operator/pull/892/files#diff-eff7340e15df41ff3a96507abf744eb5fea3f670fbb381edc624b52cda6b1b79R13-R17) we give. The golden image steps are somewhat complex as is so I'd like to avoid sending them off this page.